### PR TITLE
Seconds support

### PR DESCRIPTION
**Pull request by Jarsefax**

### CODE REVIEW

File: GoldenFox/ScheduleParser.cs [hunk 1]
```diff
@@ -18,13 +30,15 @@ public class ScheduleParser
         public ScheduleParser()
         {
             _tokenizer = new Tokenizer();
-            Add("every", "day", "month", "week", "year", "hour", "minute",
+            Add("every", "day", "month", "week", "year", "hour", 
+                "minute", "minutes", "m,", "second", "seconds",
```
Comment (mattiasnordqvist) [inline GoldenFox/ScheduleParser.cs]: what is "m,"?

File: GoldenFox/ScheduleParser.cs [hunk 2]
```diff
@@ -18,13 +30,15 @@ public class ScheduleParser
...
                 "at", "@", ":",
                 "st", "nd", "rd", "th",
                 "last",
-                "s",
+                "s", "m",
                 "and",
-                "between");
+                "between",
+                "hh:mm:", "hh:", "mm:");
```
Comment (mattiasnordqvist) [inline GoldenFox/ScheduleParser.cs:41]: Could this be replaced by just "hh" and "mm"? ":" is already in the token list

File: GoldenFox/ScheduleParser.cs (hunk 1 above)
Comment (Jarsefax) [inline GoldenFox/ScheduleParser.cs]: short for minute as s is shirt for second/seconds

File: GoldenFox/ScheduleParser.cs (hunk 2 above)
Comment (Jarsefax) [inline GoldenFox/ScheduleParser.cs:41]: I don't think so if the new at/@/between syntax ( "hh:xx"/"hh:xx:yy", "hh:mm:xx"/"mm:xx" & "hh:mm(:ss)" ) in the diagram is to work.

File: GoldenFox/ScheduleParser.cs (hunk 1 above)
Comment (mattiasnordqvist) [inline GoldenFox/ScheduleParser.cs]: "m" is short for minute, not "m," ?

Comment (Jarsefax) [inline GoldenFox/ScheduleParser.cs]: Good catch, don't know why that was there.

**Pull request closed without merging**